### PR TITLE
Add "inline-comment-spacing" lint rule

### PR DESCRIPTION
**Pull request by andy-ms**
This would check that we write `f(/*argName*/ true)` and not `f(/* argName */ true)`.
I'd also like to lint for spacing on statement comments like:
```ts
/* prefer this? */
foo();

/*prefer this?*/
foo();
```
But we don't have a consistent style here -- thoughts?

### CODE REVIEW

**Comment by billti** [comment]
Personally, I find it far more readable with the spaces providing some separation. Is there a general community default/agreement on this (i.e. linter defaults)? Or just personal preference?

**Comment by andy-ms** [comment]
We get 5000 lines of errors if I invert the test condition, so we've been preferring `/*comment*/` within argument lists for a while. Note that this rule doesn't effect comments on statements.

File: scripts/tslint/inlineCommentSpacingRule.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        if (trailing) {
+            for (const range of trailing) {
+                cb(node, range);
+            }
+        }
+        node.forEachChild(recur);
+    });
+}
+
+
```
Comment (aozgaa) [inline scripts/tslint/inlineCommentSpacingRule.ts]: nit: extra trailing lines?

File: tslint.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
         "indent": [true,
             "spaces"
         ],
+        "inline-comment-spacing": true,
```
Comment (aozgaa) [inline tslint.json]: Maybe we should have a separate rule for `"inline-callExpressionArgument-comment-spacing"`? Or maybe name it analogously to `"boolean-trivia"`? eg: `"boolean-trivia-spacing"`?

Comment (andy-ms) [inline tslint.json]: How about renaming `boolean-trivia` to `literal-argument-comment` and `inline-comment-spacing` to `argument-comment-spacing`?

Comment (aozgaa) [inline tslint.json]: Are `scripts/tslint` rules accessible outside our repo? Are rules in this directory a sort of test-bed for adding rules to tslint itself?

If these are only used by us, then that works for me. We probably shouldn't rename these if someone else is using the `boolean-trivia` rule.

Comment (andy-ms) [inline tslint.json]: These are our own internal rules. Some of them might be suitable to submit to tslint itself, if you want to.

**Comment by mhegazy** [comment]
Do we need this new rule? seems like a lot of work, and limited value in my opinion.

**Comment by mhegazy** [comment]
Do not think we need this one. closing for now, please reopen if you feel otherwise.

**Pull request closed without merging**